src/core/toga/app.py

```diff
@@ -415,6 +416,33 @@ def startup(self):
 
         self.main_window.show()
 
+    def about_command(self, widget):
+        """Default implementation of the "About" command.
+        This can be overridden in inherited App classes."""
```

Comment:
Given the discussion about platform-specific about dialogs, this probably needs to be abstracted to an API on the implementation layer.